### ISSUE TITLE
Copy of Accessibility Fixes Student Level AA Standards 

### ISSUE THREAD

**Issue by michesmith** [bcgov/SIMS#3005]
**Acceptance Criteria**

Detailed Spreadsheet for Student s inserted inline and also attached here: [2024_Student Application_Audit.xlsx](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBN0xsQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--578821d4addd12c1babcafba5c861e1f5456bc9c/2024_Student%20Application_Audit.xlsx)

**Student** 
Adjust the following Accessibility Criteria for the following  pages 
- [ ]  Login 1 and 2
- [ ]  Profile 
- [ ] Home page 
- [ ] Start new application 
- [ ] My applications 
- [ ] File uploader 
- [ ] Request a change 
- [ ] Overawards balance 

**1.0 Perceivable**
 1.4 Distinguishable
- [ ] Contrast Minimum- The visual presentation of text and images of text has a contrast ratio of at least 4.5:1, except in the cases of large text, decorative text, inactive text, invisible text, or logotypes. 
<img width="2000" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNTNsQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--f6365e02b807d7dd5ddfc669e58e1f50321635ee/image.png" alt="image.png" />

- [ ] Non-Text Contrast -The visual presentation of user interface components and graphical objects have a contrast ratio of at least 3:1 against adjacent colour(s)
<img width="2000" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNTdsQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--ae599f64a201ebcc3c45b8364982b7a9438679a2/image.png" alt="image.png" />

- [ ] Content on Hover or Focus- If pointer triggers additional content, cursor should be able to move over the additional content without it disappearing.
 <img width="2000" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNkRsQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--5dea0d2a55be5b983f097f4f2641f08a90e54862/image.png" alt="image.png" />

**2.0 Operable**
2.4 Navigable
- [ ] Focus Visible- Any keyboard operable user interface has a mode of operation where the keyboard focus indicator is visible
<img width="2000" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNlBsQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--eff3343a06df1f0ed52c336b78504370731e4aff/image.png" alt="image.png" />

 
2.5 Input Modalities
- [ ] Target Size (Minimum) -The size of the target for pointer inputs is at least 24 by 24 CSS pixels, except in specific circumstances.
<img width="2000" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNlhsQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--7f736e7a81d6610f5c5d5c89f954c2f862c6d7bb/image.png" alt="image.png" />

**4.0 Robust**
4.1 Compatible
- [ ] Status Massages -In content implemented using markup languages, status messages can be programmatically determined through role or properties such thaat they can be presented to the user by assistive technologies without receiving focus.
<img width="2000" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNnJsQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--897163bb2c2292d44ef3aae820902db03c2939d2/image.png" alt="image.png" />

**3.0 Understandable**
3.2 Predictable
- [ ] Consistent Navigation
